command/fmt_test.go

```diff
@@ -18,80 +19,115 @@ func TestFmtCommand(t *testing.T) {
 
 	const inSuffix = ".in.hcl"
 	const expectedSuffix = ".out.hcl"
-	tests := [][]string{
-		{"nomad", "-check"},
-		{"nomad", ""},
-		{"job", "-check"},
-		{"job", ""},
-	}
+
 	tmpDir := t.TempDir()
```

Comment:
(see later suggestion) Should create the tempdir in the test case if we're parallelizing them
```suggestion
```